content/en/blog/2020/istiocon-2021/istiocon-2021.md

```diff
@@ -0,0 +1,16 @@
+# Join us for the first IstioCon in 2021!
+
+IstioCon 2021 will be the inaugural conference for Istio, the industry's most popular service mesh. In its inaugural year, IstioCon will be 100% virtual, connecting community members across the globe with Istio's ecosystem. This conference will take place at the end of February. 
+
+![/macruzbar/istio.io/content/en/blog/2020/istioconlogo.jpg]
```

Comment:
Put a copy of the image in `/blog/2020/istiocon-2021/istioconlogo.jpg` and then reference it here:
```suggestion
{{< image width="75%"
    link="./istioconlogo.jpg"
    alt="IstioCon logo"
    >}}
```
You'll need to look at the preview to decide what `image width="75%"` is appropriate.

Reply:
I changed the syntax. I can see the file added in the folder on my repo here: https://github.com/macruzbar/istio.io/tree/cf63f6fc1d809cb8668a830e67c69735844f82dd/content/en/blog/2020/istiocon-2021 but I'm not sure it is part of the batch I submitted on this PR.